analyzer/analyzer.go

```diff
@@ -274,15 +274,15 @@ if (plugins.hasPlugin('com.android.application')) {
 	if strings.HasPrefix(compileSDKVersion, "android-") {
 		_, err = version.NewVersion(strings.TrimPrefix(compileSDKVersion, "android-"))
 		if err != nil {
-			return "", "", fmt.Errorf("failed to parse compileSDKVersion (%s), error: %s", compileSDKVersion, err)
+			return "", "", fmt.Errorf("Failed to parse compileSDKVersion (%s), error: %s", compileSDKVersion, err)
```

Comment:
please do not return with capitalised error. The caller function should handle to print the error message with the correct capital.

In this case `analyzeWithGradlew` function is used only in `NewProjectDependencies` function, `NewProjectDependencies` function is called in the `main` function and the error message is wrapped with a capitalised message.